[Enhancement] - ServiceNow - Non-Interactive OAuth2 for v2 Integration (#42975)
demisto/content: Packs/ServiceNow/Integrations/ServiceNowEventCollector/ServiceNowEventCollector_description.md

#### OAuth 2.0 Authorization
To use OAuth 2.0 authorization:
1. Log in to your ServiceNow instance and create an endpoint for Cortex XSIAM to access your instance. For more information, see [Snow OAuth](https://docs.servicenow.com/bundle/orlando-platform-administration/page/administer/security/task/t_CreateEndpointforExternalClients.html) .
2. Click the lock next to the Client Secret to reveal it.
3. Copy the `Client Id` and `Client Secret` into the `ClientID` and `Client Secret` fields of the instance configuration. The `Client Id` and `Client Secret` were automatically generated when you created the endpoint.

> 4. Enter the ServiceNow account's `Username` and `Password` in the instance configuration. Since these fields are required, the integration will automatically perform the initial OAuth login on first use and automatically renew the refresh token when it expires — no manual login command is needed.

Reviewer: ```suggestion
4. Enter the ServiceNow account **Username** and **Password** in the integration instance configuration. The integration will automatically perform the initial OAuth login on first use and automatically renew the refresh token when it expires.
```

> 5. Select the `Use OAuth Login` checkbox and click `Done`.

Reviewer: ```suggestion
5. Select the Use **OAuth Login** checkbox and click **Done**.
```
